doc/api/animation_api.rst

```diff
@@ -55,7 +55,7 @@ performance), to be non-blocking, not repeatedly start/stop the GUI
 event loop, handle repeats, multiple animated axes, and easily save
 the animation to a movie file.
 
-'Blitting' is a `old technique
+'Blitting' is a `standard technique
```

Comment:
Since this example code cannot really be actually used (it would fail if the screen is resized, for example), and now we explain blitting much more in-depth in a separate tutorial, what is the purpose of having a more complicated version of what amounts to pseudocode here? 

Either way, we should make "handle 'blitting'" above a link to the blitting tutorial. But once that link is there, does the average `Animation` user gain anything except for an unsolicited lesson on rendering by being shown an outline of how blitting works? 

I think if the point is to convince people to set `blit=True`, then we should just mention that we always suggest doing so, and to see the tutorial for details on the implementation if you're interested.

Reply:
I think the goal of the mini-blitting intro here is to justify why the signature of the callback in `FuncAnimation` is what it is.

I don't disagree that this should be re-written, but would rather not do it in this (already pretty old) PR.

Reply:
The other ways of doing the callback function should also be written out...

Reply:
Yeah, I guess my top-level comment is that I don't feel like that sections adds anything that you wouldn't also get if you just changed 

```
If blit == True, func must return an iterable of all artists that were modified or created.
```

to 

```
If blit == True, then only modified artists are re-drawn, and func must return an iterable of all artists that were modified or created.
```

And, frankly, the docs would be equally useful with that existing section totally removed now that you have a blitting tutorial.

Now that you've added a blitting tutorial, it feels like it detracts from the docs by serving as a lot of unnecessary "noise" to describe something that the user doesn't even need to understand to correctly use the function.

Reply:
I'm going to stick to doing this removal in a second pass.